kernel/Kconfig

```diff
@@ -554,6 +554,13 @@ config EVENTS
 	  Note that setting this option slightly increases the size of the
 	  thread structure.
 
+config PIPES
+	bool "Pipe objects"
```

Comment:
Can we get a "depends on !KERNEL_COHERENCE" here, or a `BUILD_ASSERT(!defined(CONFIG_KERNEL_COHERENCE));` in code somewhere, or some other equivalent validation (with an explanation somewhere adjacent) just so no one trips over the issue accidentally?